src/main/java/Deadline.java

```diff
@@ -0,0 +1,49 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+public class Deadline extends Task{
+
+    private String deadline;
+    private LocalDate deadlineDate;
+    private boolean isParsed = false;
+
+    public Deadline(String name, String deadline, boolean isDone){
+
+        super(name, isDone);
+        this.deadline = deadline;
+        try {
+            this.deadlineDate = LocalDate.parse(deadline);
+            this.isParsed = true;
+        } catch(DateTimeParseException e){
+
+            System.out.println("Date cannot be parsed, stored as String instead.");
+        }
+    }
+
+    @Override
+    public String toString(){
+        if(isParsed){
+
```

Comment:
I don't think the coding standard mentions anything about leaving blank lines after the declaration of a method or class, but is there a reason why you do this? The toString method above does not have the blank line for example.

In all the other classes I also see many blank lines in random spots... Not sure if they are placed there on purpose but I can't really understand the logic of having those spaces at those spots.